lib/matplotlib/figure.py

```diff
@@ -2986,7 +2986,8 @@ def set_canvas(self, canvas):
 
     @_docstring.interpd
     def figimage(self, X, xo=0, yo=0, alpha=None, norm=None, cmap=None,
-                 vmin=None, vmax=None, origin=None, resize=False, **kwargs):
+                 vmin=None, vmax=None, colorizer=None, origin=None, resize=False,
+                 **kwargs):
```

Comment:
This one isn't keyword-only yet.

Reply:
I added a `*`
```py
    def figimage(self, X, xo=0, yo=0, alpha=None, norm=None, cmap=None,
                 vmin=None, vmax=None, origin=None, resize=False, *,
                 colorizer=None, **kwargs):
```
also to the typing figure.pyi, now reads:
```py
        resize: bool = ...,
        *,
        colorizer: Colorizer | None = ...,
        **kwargs
    ) -> FigureImage: ...
```